modules/making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,6 @@
+[preface]
```

Comment:
I noticed this in the other PR but what is the `[preface]` tag used for?

Reply:
So, I'm not sure if this works with PV2. If it does - preface allows you have a 'chapter' without a number.  So you can see it here: https://access.redhat.com/documentation/en-us/openjdk/11/html/configuring_openjdk_11_on_rhel_with_fips/index